GDevelop.js/scripts/generate-dts.mjs

```diff
@@ -175,7 +175,9 @@ for (const [_, interfaceName, interfaceCode] of bindingsFile.matchAll(
 
     if (isAttribute) {
       const attributeName = Parser.readUntil(';');
-      attributes.push(`${attributeName}: ${returnType};`);
+      attributes.push(
+        `${attributeName}${optionalReturn ? '?' : ''}: ${returnType};`
```

Comment:
I don't think this makes sense, class properties cannot be optional in c++ after all